docs/plugins.md

```diff
@@ -234,6 +234,16 @@ Ready-to-go basemaps, with little or no configuration at all.
 			<a href="https://github.com/rkaravia">Roman Karavia</a>
 		</td>
 	</tr>
+  <tr>
```

Comment:
Please use tabs for indent

Reply:
Please use tabs for indent

Reply:
Looks like this was just for one single line. This has now been updated.